test/apiv2/10-images.at

```diff
@@ -45,6 +45,11 @@ t POST "images/create?fromImage=alpine" 200 .error~null .status~".*Download comp
 
 t POST "images/create?fromImage=alpine&tag=latest" 200
 
+# 10977 - handle platform parameter correctly
+t POST "images/create?fromImage=alpine&platform=linux/arm64" 200
+t GET  "images/alpine/json" 200
+is "$(echo $output | jq .Architecture)" "\"arm64\"" "platform parameter works"
```

Comment:
Can you please use the following like the other tests:
```suggestion
t GET  "images/alpine/json" 200 \
  .Architecture=arm64
```
This looks much better in the test log.